common/src/main/java/com/wynntils/gui/widgets/TextInputBoxWidget.java

```diff
@@ -16,17 +16,15 @@
 import net.minecraft.client.gui.Font;
 import net.minecraft.client.gui.components.AbstractWidget;
 import net.minecraft.client.gui.narration.NarrationElementOutput;
+import net.minecraft.client.gui.screens.Screen;
 import net.minecraft.client.resources.sounds.SimpleSoundInstance;
 import net.minecraft.network.chat.Component;
 import net.minecraft.network.chat.TextComponent;
 import net.minecraft.sounds.SoundEvents;
 import org.lwjgl.glfw.GLFW;
 
-// FIXME: This is a very basic text box.
+// FIXME: Add selection support to this class to be a fully working text box
 public class TextInputBoxWidget extends AbstractWidget {
-    protected long lastKeyDownCheck = 0;
-    protected long firstKeyDown = 0;
```

Comment:
These were not used for a while. I should have removed them when I added the repeating key support for screens.